### ISSUE TITLE
Exception in plugin flutter 47.0.2

### ISSUE THREAD

**Issue by AliHG** [flutter/flutter-intellij#4665]
## Steps to Reproduce
opening a f lutter project 
## What happened

I had this error after updating the flutter plugin on android studio

## Version information

Android Studio `4.0` • Flutter plugin `io.flutter 47.0.2` • Dart plugin `193.7361`

Error getting Flutter sdk information.

## Exception

null

```
java.lang.NullPointerException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at io.flutter.android.AndroidModuleLibraryManager$EmbeddedAndroidProject.initPre41(AndroidModuleLibraryManager.java:355)
	at io.flutter.android.AndroidModuleLibraryManager.doGradleSync(AndroidModuleLibraryManager.java:216)
	at io.flutter.android.AndroidModuleLibraryManager.update(AndroidModuleLibraryManager.java:93)
	at io.flutter.android.AndroidModuleLibraryManager.updateAndroidLibraries(AndroidModuleLibraryManager.java:207)
	at com.intellij.openapi.project.DumbServiceImpl.lambda$smartInvokeLater$7(DumbServiceImpl.java:460)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:309)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:441)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:908)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:781)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:424)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:698)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:423)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

## Version info

[√] Flutter (Channel stable, v1.17.4, on Microsoft Windows [Version 10.0.19041.329], locale en-US)
    • Flutter version 1.17.4 at C:\src\flutter
    • Framework revision 1ad9baa8b9 (10 days ago), 2020-06-17 14:41:16 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at D:\Programming\programs\android-sdk
    • Platform android-30, build-tools 30.0.0
    • ANDROID_HOME = D:\Programming\programs\android-sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 47.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
[√] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 10 (API 29) (emulator)

• No issues found!

**Comment by devoncarew** [comment]
cc @stevemessick for the `AndroidModuleLibraryManager` exception in AS 4.0.

**Comment by Gag1011** [comment]
I have the same error

**Comment by AliHG** [comment]
Fixed with the new flutter plugin v 47.1.2
